package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "api-server",
+  "version": "1.0.0",
+  "description": "Built a REST API using Express, by creating a proper series of endpoints that perform CRUD operations on a database, using the REST standard",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest --verbpse - coverage",
```

Comment:
You have a spelling error here, it should be:

`"test": "jest --verbose --coverage"`